### ISSUE TITLE
onCropImage is not working as expected in web

### ISSUE THREAD

**Issue by umamaheswari2023** [icapps/flutter-custom-image-crop#40]
I have implemented the plugin for handling the profile image crop and zoom functionality. If i select a new image and try to zoom in and do the crop, but it doesn't returned the cropped image.

Uint8List binaryImage = (await viewModel.cropController.onCropImage())?.bytes;

**Comment by ikbendewilliam** [comment]
Does it return null or an empty image? 
Do you wait for it to be loaded or crop while loading?
Have you tried running the example?

**Comment by umamaheswari2023** [comment]
Yes I have tried with your example, but I found one scenario in which the onCropImage is not working. If I am picking the image from image_picker and set the Uint8List value to the CustomImageCrop widget, then this is not working. because the controller is not updated to the newly added image. 

Could you please suggest a solution for updating the controller on selecting the image dynamically?

**Comment by ikbendewilliam** [comment]
Ah, that is correct. We should add a `didUpdateWidget(oldWidget)` method where we can then update the controller. 
Feel free to create a PR for this! (I can maybe fix it somewhere this week, but it might take a few days).

**Comment by umamaheswari2023** [comment]
Is it fine to call the _getImage method under the didUpdateWidget function?

**Comment by ikbendewilliam** [comment]
You can definetly try adding this and seeing if this fixed the issue. 
That is what we're doing in the `didChangeDependencies` as well. Can you share a piece of your code where you load the image?

**Comment by ikbendewilliam** [comment]
@umamaheswari2023 I've created a PR for this #41, can you double check that this fixes your issue? You can use it by adding this to your `pubspec.yaml`

```yaml
dependency_overrides: 
  custom_image_crop:
    git:
      url: https://github.com/icapps/flutter-custom-image-crop
      ref: 7a04a982f4b565c08870c844399398d5e04aacb0
```

**Comment by umamaheswari2023** [comment]
Hi @ikbendewilliam  This PR fixed my issue. Can you please merge this PR, So that I can use the stable version.

**Comment by ikbendewilliam** [comment]
Perfect, yes I'll merge and release asap

**Comment by ikbendewilliam** [comment]
Released as 0.0.10
